tempesta_fw/procfs.h

```diff
@@ -49,20 +48,30 @@ typedef struct {
  * @rx_bytes		- The number of bytes received from peers and
  *			  processed by Tempesta.
  */
-typedef struct {
-	u64	rx_messages;
-	u64	msgs_forwarded;
-	u64	msgs_fromcache;
-	u64	msgs_parserr;
-	u64	msgs_filtout;
-	u64	msgs_otherr;
+#define TFW_STAT_COMMON							\
+	u64	rx_messages;						\
+	u64	msgs_forwarded;						\
+	u64	msgs_parserr;						\
+	u64	msgs_filtout;						\
+	u64	msgs_otherr;						\
+	u64	conn_attempts;						\
+	u64	conn_established;					\
+	u64	conn_disconnects;					\
+	u64	rx_bytes;
 
-	u64	conn_attempts;
-	u64	conn_established;
-	u64	conn_disconnects;
+typedef struct {
+	TFW_STAT_COMMON;
+} TfwSrvStat;
 
-	u64	rx_bytes;
-} TfwPeerStat;
+/*
+ * @msgs_fromcache	- The number of messages served from cache.
+ * @online		- The number of clients online.
+ */
+typedef struct {
+	TFW_STAT_COMMON;
+	u64	msgs_fromcache;
+	u64	online;
```

Comment:
This is online _clients_, not connections. And we have client specific structure, so `clnt.clinet_online` doesn't have too much sense.